### ISSUE TITLE
Missing accessibilityIdentifier on keyboard's toolbar only on decimal pad

### ISSUE THREAD

**Issue by palla89** [hackiftekhar/IQKeyboardManager#1682]
**Describe the bug**
The Next button of the keyboard's toolbar is ignoring / missing the accessibilityLabel property like toolbarNextBarButtonItemAccessibilityLabel. 

**To Reproduce**
I'm writing XCode UI Tests and I can't reach the "next" button on the keyboard decimal pad's toolbar. I can instead access it correctly on the standard keyboard. Am I missing something?

**Expected behavior**
Just like I set the toolbarNextBarButtonItemAccessibilityLabel, I'd expect to have it working also on the decimal's pad.

### Versions
**Xcode:** 11.3
**Mac OS:** 10.15.2
**Simulator/Device:** 13.3
**Simulator/Device Name:** iPhone 11 Pro Max
**Library Version:** 6.5.4

**Comment by hackiftekhar** [comment]
Just to let you know that the library do not use different code logic for iPad and iPhone. It uses the same accessibility label for iPad too.

**Comment by palla89** [comment]
Hi! Yes I mean another thing, the accessibility identifier for this:

![D00C4E22-B11F-42B1-B257-8F6BBF491F90](https://user-images.githubusercontent.com/12301205/72221589-9b7ac880-355c-11ea-984e-721dde9d13b3.jpeg)

Maybe it’s more clear now :)

**Comment by hackiftekhar** [comment]
I'm sure we have these exposed properties
toolbarPreviousBarButtonItemAccessibilityLabel
toolbarNextBarButtonItemAccessibilityLabel
toolbarDoneBarButtonItemAccessibilityLabel

and you could set it to reach the previous/next and done buttons.

**Comment by blyscuit** [comment]
I also tried setting 
```
        IQKeyboardManager.shared.toolbarDoneBarButtonItemAccessibilityLabel =
            AccessibilityIdentifiers.Keyboard.done.rawValue
```
but the `label` seems to be overrided.

<img width="776" alt="Screen Shot 2021-04-05 at 16 00 36" src="https://user-images.githubusercontent.com/6356137/113557539-5e4e9a80-9628-11eb-86c0-cee9cb969368.png">
